src/docfx/SubCommands/DocumentBuilderWrapper.cs

```diff
@@ -190,9 +190,12 @@ private static IEnumerable<Assembly> LoadPluginAssemblies(string pluginDirectory
 
         private static IEnumerable<DocumentBuildParameters> ConfigToParameter(BuildJsonConfig config, TemplateManager templateManager, ChangeList changeList, string baseDirectory, string outputDirectory, string templateDir)
         {
-            var parameters = new DocumentBuildParameters();
-            parameters.OutputBaseDir = outputDirectory;
-            parameters.ForceRebuild = config.Force ?? false;
+            var parameters = new DocumentBuildParameters
```

Comment:
>DocumentBuildParameters [](start = 33, length = 23)

can you confirm adding ForcePostProcess here won't impact incremental build (compile + link)?